# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1264682608 18000
#      Thu Jan 28 07:43:28 2010 -0500
# Node ID f3c1fd990f3f786474b6edeacf88d1bb7c2c18d4
# Parent  60d382fd0063d8dfafd2ce94120bf586fb8ffafe
note some deprecated functions are still in Matlab, so don't remove with other functions deprecated in 3.0.0

diff --git a/doc/interpreter/obsolete.txi b/doc/interpreter/obsolete.txi
--- a/doc/interpreter/obsolete.txi
+++ b/doc/interpreter/obsolete.txi
@@ -48,17 +48,16 @@ as the obsolete function, but should pro
 @item @code{binomial_inv} @tab @code{binoinv} @tab 3.4.0
 @item @code{binomial_pdf} @tab @code{binopdf} @tab 3.4.0
 @item @code{binomial_rnd} @tab @code{binornd} @tab 3.4.0
 @item @code{chisquare_cdf} @tab @code{chi2cdf} @tab 3.4.0
 @item @code{chisquare_inv} @tab @code{chi2inv} @tab 3.4.0
 @item @code{chisquare_pdf} @tab @code{chi2pdf} @tab 3.4.0
 @item @code{chisquare_rnd} @tab @code{chi2rnd} @tab 3.4.0
 @item @code{clearplot} @tab @code{clf} @tab 3.4.0
-@item @code{clg} @tab @code{clf} @tab 3.4.0
 @item @code{com2str} @tab @code{num2str} @tab 3.4.0
 @item @code{exponential_cdf} @tab @code{expcdf} @tab 3.4.0
 @item @code{exponential_inv} @tab @code{expinv} @tab 3.4.0
 @item @code{exponential_pdf} @tab @code{exppdf} @tab 3.4.0
 @item @code{exponential_rnd} @tab @code{exprnd} @tab 3.4.0
 @item @code{f_cdf} @tab @code{fcdf} @tab 3.4.0
 @item @code{f_inv} @tab @code{finv} @tab 3.4.0
 @item @code{f_pdf} @tab @code{fpdf} @tab 3.4.0
@@ -81,17 +80,16 @@ as the obsolete function, but should pro
 @item @code{is_list} @tab @code{islist} @tab 3.4.0
 @item @code{is_matrix} @tab @code{ismatrix} @tab 3.4.0
 @item @code{is_scalar} @tab @code{isscalar} @tab 3.4.0
 @item @code{is_square} @tab @code{issquare} @tab 3.4.0
 @item @code{is_stream} @tab @code{isstream} @tab 3.4.0
 @item @code{is_struct} @tab @code{isstruct} @tab 3.4.0
 @item @code{is_symmetric} @tab @code{issymmetric} @tab 3.4.0
 @item @code{is_vector} @tab @code{isvector} @tab 3.4.0
-@item @code{isstr} @tab @code{ischar} @tab 3.4.0
 @item @code{lognormal_cdf} @tab @code{logncdf} @tab 3.4.0
 @item @code{lognormal_inv} @tab @code{logninv} @tab 3.4.0
 @item @code{lognormal_pdf} @tab @code{lognpdf} @tab 3.4.0
 @item @code{lognormal_rnd} @tab @code{lognrnd} @tab 3.4.0
 @item @code{meshdom} @tab @code{meshgrid} @tab 3.4.0
 @item @code{normal_cdf} @tab @code{normcdf} @tab 3.4.0
 @item @code{normal_inv} @tab @code{norminv} @tab 3.4.0
 @item @code{normal_pdf} @tab @code{normpdf} @tab 3.4.0
@@ -100,17 +98,16 @@ as the obsolete function, but should pro
 @item @code{pascal_inv} @tab @code{nbininv} @tab 3.4.0
 @item @code{pascal_pdf} @tab @code{nbinpdf} @tab 3.4.0
 @item @code{pascal_rnd} @tab @code{nbinrnd} @tab 3.4.0
 @item @code{poisson_cdf} @tab @code{poisscdf} @tab 3.4.0
 @item @code{poisson_inv} @tab @code{poissinv} @tab 3.4.0
 @item @code{poisson_pdf} @tab @code{poisspdf} @tab 3.4.0
 @item @code{poisson_rnd} @tab @code{poissrnd} @tab 3.4.0
 @item @code{polyinteg} @tab @code{polyint} @tab 3.4.0
-@item @code{setstr} @tab @code{char} @tab 3.4.0
 @item @code{struct_contains} @tab @code{isfield} @tab 3.4.0
 @item @code{struct_elements} @tab @code{fieldnames} @tab 3.4.0
 @item @code{t_cdf} @tab @code{tcdf} @tab 3.4.0
 @item @code{t_inv} @tab @code{tinv} @tab 3.4.0
 @item @code{t_pdf} @tab @code{tpdf} @tab 3.4.0
 @item @code{t_rnd} @tab @code{trnd} @tab 3.4.0
 @item @code{uniform_cdf} @tab @code{unifcdf} @tab 3.4.0
 @item @code{uniform_inv} @tab @code{unifinv} @tab 3.4.0
diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,14 @@
+2010-01-28  John W. Eaton  <jwe@octave.org>
+
+	* deprecated/isstr.m, deprecated/clg.m, deprecated/setstr.m:
+	Note that Matlab still has these functions, so they probabl
+	should not be removed with other functions deprecated in 3.0.0.
+
 2010-01-27  Thorsten Meyer  <thorsten.meyier@gmx.de>
 
 	* plot/gnuplot_drawnow.m: Add "emf" to the list of enhanced
 	terminals for the gnuplot backend.
 
 2010-01-26  John W. Eaton  <jwe@octave.org>
 
 	* general/chop.m: New function.
diff --git a/scripts/deprecated/clg.m b/scripts/deprecated/clg.m
--- a/scripts/deprecated/clg.m
+++ b/scripts/deprecated/clg.m
@@ -19,16 +19,17 @@
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} clg ()
 ## This function has been deprecated.  Use clf instead.
 ## @end deftypefn
 
 ## Author: jwe
 
 ## Deprecated in version 3.0
+## Matlab still has this function, so don't remove just yet.
 
 function clg ()
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
              "clg is obsolete and will be removed from a future version of Octave, please use clf instead");
diff --git a/scripts/deprecated/isstr.m b/scripts/deprecated/isstr.m
--- a/scripts/deprecated/isstr.m
+++ b/scripts/deprecated/isstr.m
@@ -19,16 +19,17 @@
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} isstr (@var{a})
 ## This function has been deprecated.  Use ischar instead.
 ## @end deftypefn
 
 ## Author: jwe
 
 ## Deprecated in version 3.0
+## Matlab still has this function, so don't remove just yet.
 
 function retval = isstr (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
              "isstr is obsolete and will be removed from a future version of Octave, please use ischar instead");
diff --git a/scripts/deprecated/setstr.m b/scripts/deprecated/setstr.m
--- a/scripts/deprecated/setstr.m
+++ b/scripts/deprecated/setstr.m
@@ -19,16 +19,17 @@
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} setstr (@var{s})
 ## This function has been deprecated.  Use char instead.
 ## @end deftypefn
 
 ## Author: jwe
 
 ## Deprecated in version 3.0
+## Matlab still has this function, so don't remove just yet.
 
 function retval = setstr (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
              "setstr is obsolete and will be removed from a future version of Octave; please use char instead");
